src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingService.kt

```diff
@@ -217,6 +219,7 @@ class SentencingService(
       }
     }
 
+  // TODO not currently updating the EXISTING offendercharge - eg with new offenceresultcode
   @Audit
```

Comment:
will need to update the underlying offender charge as well as creating the court event charge